src/triggers/createPullRequestForJiraIssue.ts

```diff
@@ -134,7 +136,7 @@ export const createPullRequestForJiraIssue = async (
       info(
         `Issue ${issue.key} belongs to epic ${epic.key} - creating an Epic pull request.`
       )
-      const epicPr = await createEpicPullRequest(epic, issue.fields.repository)
+      const epicPr = await createEpicPullRequest(epic, repository)
```

Comment:
As above - let's set `repository` to `issue.fields.repository` at the beginning if `repository` has not been passed, to keep backwards-compatibility.